tools/build/package.json

```diff
@@ -39,15 +39,15 @@
     "mocha-junit-reporter": "^2.0.2",
     "rimraf": "^3.0.2",
     "tree-kill": "^1.2.2",
-    "typedoc": "^0.23.28",
+    "typedoc": "^0.25.8",
```

Comment:
fyi @ben-polinsky

Reply:
docs validation is now passing.  thanks @ben-polinsky!!